src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,12 +34,624 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+                    context: contentDbContext,
+                    userService: userService.Object);
+
+                var result = await publicationService
+                    .ListPublications(permissions: false, topic.Id);
+
+                var publicationViewModelList = result.AssertRight();
+
+                Assert.Equal(3, publicationViewModelList.Count);
+
+                Assert.Equal(publication1.Id, publicationViewModelList[0].Id);
+                Assert.Equal(publication1.Title, publicationViewModelList[0].Title);
+
+                Assert.Equal(publication2.Id, publicationViewModelList[1].Id);
+                Assert.Equal(publication2.Title, publicationViewModelList[1].Title);
+
+                Assert.Equal(publication3.Id, publicationViewModelList[2].Id);
+                Assert.Equal(publication3.Title, publicationViewModelList[2].Title);
+            }
+        }
+
+        [Fact]
+        public async Task ListPublications_CannotViewAllPublications_NoTopic()
+        {
+            var user = new User { Id = Guid.NewGuid(), };
+
+            var publication1 = new Publication
+            {
+                Title = "publication1",
+                Topic = new Topic { Theme = new Theme(), },
+            };
+
+            var publication2 = new Publication
+            {
+                Title = "publication2",
+                Topic = new Topic { Theme = new Theme(), },
+            };
+
+            var publication3 = new Publication
+            {
+                Title = "publication3",
+                Topic = new Topic { Theme = new Theme(), },
+            };
+
+            var userService = new Mock<IUserService>(Strict);
+
+            userService.Setup(s => s.GetUserId()).Returns(user.Id);
+            userService.Setup(s => s.MatchesPolicy(CanAccessSystem)).ReturnsAsync(true);
+            userService.Setup(s => s.MatchesPolicy(CanViewAllPublications)).ReturnsAsync(false);
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.Publications.AddRangeAsync(publication1, publication2, publication3);
+                await contentDbContext.UserPublicationRoles.AddRangeAsync(
+                    new UserPublicationRole
+                    {
+                        User = user,
+                        Publication = publication1,
+                        Role = PublicationRole.Owner,
+                    },
+                    new UserPublicationRole
+                    {
+                        User = user,
+                        Publication = publication2,
+                        Role = PublicationRole.Owner,
+                    },
+                    new UserPublicationRole
+                    {
+                        User = user,
+                        Publication = publication3,
+                        Role = PublicationRole.Owner,
```

Comment:
This test case isn't very thorough as the user has access to all publications, so there's no point of comparison to prove that publications they cannot access are being filtered out.